### PR TITLE
<Card/> - add withShadow prop

### PR DESCRIPTION
**Pull request by yogevitz**
### 🔦 Summary

add a boolean `withShadow` prop (default: false) to the `<Card/>` component

### ✅ Checklist

- [x] 👨‍💻 API change is approved by the UI Infra Developers @EgoziE 
- [x] 👨‍🎨 UX change is approved by the Design System UX @milkyfruit 
- 📚 Change is documented
  - [x] API description
- 🔬 Change is tested
  - [x] Component

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6257

**Review by EgoziE — CHANGES_REQUESTED** [review]
Looks good, just please add one visual test.
To `src/Card/Card.visual.js`

File: src/Card/Card.st.css [hunk 1]
```diff
@@ -24,6 +24,10 @@
   overflow: hidden;
 }
 
+.card:withShadow {
+  box-shadow: 0 2px 4px 0 rgba(22, 45, 61, 0.12), 0 0 6px 0 rgba(22, 45, 61, 0.12);
```
Comment (milkyfruit) [inline src/Card/Card.st.css:28]: @yogevitz please use $shadow-20 instead: https://github.com/wix/wix-style-react/blob/68bc0787283669dfd2257122518cf9e7c6cf32c3/src/colors.scss#L103

File: src/Card/index.d.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ export interface CardProps {
   controls?: React.ReactNode;
   stretchVertically?: boolean;
   hideOverflow?: boolean;
+  withShadow?: boolean;
```
Comment (milkyfruit) [inline src/Card/index.d.ts:12]: @yogevitz @EgoziE Do we have some convention how boolean props should be named?
In WSR lib I found variety of solutions, but it seems usually named "verb" + "noun". Therefore I suggest to rename `withShadow` to `showShadow`.

`<TableListItem showDivider />`
`<Accordion hideShadow />`
`<Popover showArrow />`
`<ListItemSelect checkbox />`

**Review by milkyfruit — CHANGES_REQUESTED** [review]

File: src/Card/Card.st.css (hunk 1 above)
Comment (yogevitz) [reply src/Card/Card.st.css]: done :)

File: src/Card/index.d.ts (hunk 2 above)
Comment (yogevitz) [reply src/Card/index.d.ts:12]: @EgoziE wdyt?

Comment (EgoziE) [reply src/Card/index.d.ts:12]: Sure, I don't mind, showShadow is fine by me

Comment (yogevitz) [reply src/Card/index.d.ts]: done :)

**Review by EgoziE — APPROVED** [review]

**Pull request merged**